scdrs/util.py

```diff
@@ -126,6 +126,8 @@ def load_scdrs_score(
     return dict_score
 
 
+# TODO: convert species names
+# FIXIT: this line is funny: f"gene conversion from {src_species} to {dst_species} is not supported"
```

Comment:
why this line is problematic?

Reply:
I am not sure. There is one time it printed "gene conversion from {src_species} to {dst_species} is not supported" instead of the values of src_species and dst_species. But I couldn't reproduce it later. 

Now load_gs does not support src_species='human' or 'mouse'. This is the main thing I want to update.